README.md

```diff
@@ -1,19 +1,58 @@
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-[![CircleCI](https://circleci.com/gh/zenika-open-source/insights-website/tree/master.svg?style=svg)](https://circleci.com/gh/zenika-open-source/insights-website/tree/master)
-[![Netlify Status](https://api.netlify.com/api/v1/badges/44effe10-3635-426d-899d-984201fa828a/deploy-status)](https://app.netlify.com/sites/zenika-open-source-insights/deploys)
+<h1 align="center">Welcome to insights-website 👋</h1>
+<p>
+  <img src="https://img.shields.io/badge/version-0.1.0-blue.svg?cacheSeconds=2592000" />
+  <a href="https://github.com/zenika-open-source/insights-website/graphs/commit-activity">
+    <img alt="Maintenance" src="https://img.shields.io/badge/Maintained%3F-yes-green.svg" target="_blank" />
+  </a>
+  <a href="https://github.com/zenika-open-source/insights-website/blob/master/LICENSE">
+    <img alt="License: Apache 2.0" src="https://img.shields.io/badge/License-Apache 2.0-yellow.svg" target="_blank" />
+  </a>
+  <a href="https://circleci.com/gh/zenika-open-source/insights-website/tree/master">
+    <img alt="CircleCI" src="https://circleci.com/gh/zenika-open-source/insights-website/tree/master.svg?style=svg" target="_blank" />
+  </a>
+  <a href="https://app.netlify.com/sites/zenika-open-source-insights/deploys">
+    <img alt="Netlify Status" src="https://api.netlify.com/api/v1/badges/44effe10-3635-426d-899d-984201fa828a/deploy-status" target="_blank" />
+  </a>
+</p>
 
-# insights-website
+> White label website to display awesome insights about organization open source activity
```

Comment:
@zenika-open-source/insights-website should we keep "White label" while it's not one for now and maybe for a moment?
Instead it should be better to have an issue : "Make it a white label"